packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistUpdateDot.module.css

```diff
@@ -0,0 +1,8 @@
+.root {
+  /* margin left = -(6px width + 4px margin-right on update dot element) so that list items stay aligned */
+  margin-left: -10px;
```

Comment:
negative margins always feel off to me for some reason, dont want to shake up the structure that's working though

Reply:
yeah agreed... im down to do a css pass with you at some point with all this if youre game, pretty much all that is copied from previous setup